task/github.py

```diff
@@ -129,17 +130,23 @@ def __init__(self, base=None, cacher=None, repo=None):
         self.token = None
         self.debug = False
         try:
-            with open(xdg_config_home('github-token'), "r") as f:
+            with open(xdg_config_home('cockpit-dev', 'github-token'), "r") as f:
                 self.token = f.read().strip()
         except FileNotFoundError:
-            # fall back to GitHub's CLI token
             try:
-                with open(xdg_config_home("gh/config.yml")) as f:
-                    match = re.search(r'oauth_token:\s*(\S+)', f.read())
-                if match:
-                    self.token = match.group(1)
+                with open(xdg_config_home('github-token'), "r") as f:
+                    print('~/.config/github-token is deprecated.  ' +
```

Comment:
No need for the `+` -- the parser will just treat several concatenated strings as one, similar to C.